src/clover/simulation/clinic.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/python3
+########################################################################################
+# clinic.py - Health-centre clinic module                                              #
+#                                                                                      #
+# Author: Ilaria Del Frate                                                             #
+# Copyright: Ilaria Del Frate, 2022                                                    #
+# License: Open source                                                                 #
+# Most recent update: 26/07/2022                                                       #
+#                                                                                      #
+# For more information, please email:                                                  #
+#     philip.sandwell@gmail.com                                                        #
+########################################################################################
+
+"""
+clinic.py - Works out the health centre load.
+
+"""
+
+import dataclasses
+
+from typing import List
+
+import yaml
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+@dataclasses.dataclass
+class Clinic:
+    """
+    Represents a clinic.
+
+    .. attribute:: name
+        The name of the clinic.
+
+    """
+
+    name: str
+
+    # Transmission Load
+    floor_area: float  # Tells python that a clinic has an area, and it's a number
+    roof_area: float
+    surface_area_walls: float
+    surface_area_doors_windows: float
+    u_value_walls: float
+    u_value_windows_doors: float
+    u_value_floor: float
+    u_value_roof: float
+    inside_ideal_temperature: float
+    # outside_temperature: float
+
+    # Internal heat load - people
+    people_1: float
+    people_2: float
+    heat_loss_people_1: float
+    heat_loss_people_2: float
+    start_time_people_1: List[int]
+    end_time_people_1: List[int]
+    start_time_people_2: List[int]
+    end_time_people_2: List[int]
+
+    # Internal heat load - Lighting
+    lamps_internal: float
+    lamps_external: float
+    start_time_lamps: List[int]
+    end_time_lamps: List[int]
+    time_lamps_external: float
+    wattage_lamps: float
+
+    # Equipment heat load
+    fridge_wattage: float
+    time_fridge: float
+
+    # Infiltration loads
+    changes: float
+    volume_cold_store: float
+    energy_new_air: float
+    start_time_infiltration: List[int]
+    end_time_infiltration: List[int]
+
+    run_hours: float
+
+
+MY_PATH = "C:/Users/Ilaria/CLOVER/locations/Bahraich/inputs/simulation/"
```

Comment:
## :bug: Hardcoding
In general, and Paul will be tired at the amount of I've gone on about this, you don't want to "hard code" things.

### What is hardcoding?
Hardcoding is putting information straight into Python files. The way that CLOVER is set up is that all of the input information that you need should sit in the `locations/<my_location>` directory, and CLOVER should be able to run over any location, with no information directly written into the code about your location, your clinic etc.

### Why is it bad?
Generally, it means that the code becomes very temperamental, with slight changes, e.g., from Windows to Linux or Mac, will break the code. Here, by using the full path to the file on your windows machine, you've effectively said "this code can only be run on my computer, using my input file." What this means is that the code will only run on your computer, and it will only run so long as this file exists, and is exactly where you put it.

### What to do instead?
You generally want all the information to be either inputted on the command-line interface, using arguments such as `-l Bahraich` (which you'll hopefully be familiar with from running CLOVER?), or in the input files. For testing the code, you want everything in the `if __name__ == "__main__"` block at the bottom, rather than writing strings into the code like this :smile:

## 🔍 This file?
For the purposes of us testing the code, having a file path like this is probably fine... however, it means that an exact copy of your location, how you've got it coded, including all your input files, needs to either be emailed over to us as a `.zip` file, or attached to this pull request (you can upload a file I believe as a comment).